.github/.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "tabWidth": 2,
+  "useTabs": false
+}
+
```

Comment:
What's this file for? We have a `.editorconfig` BTW with specific configurations for the repo

Reply:
Fair point, this was added because prettier was malforming the YAML, likely because the `.editorconfig` is set to defaults which violate the linter.